### DISCUSSION TITLE
How to edit the config file from v8 to v9

### DISCUSSION THREAD

**Discussion by c0sc0s** [eslint/eslint#18283] [Q&A] [answered]
Today, i want to upgrade the eslint from `v8` to `v9` 
But i got a error with 'can not find config file' .

I used ` .eslintignore` and `.eslintrc.cjs` for my project.
And how can i format it for the new version, thx.

**Comment by fasttime** [comment, marked as answer]
There is a migration guide here: https://eslint.org/docs/latest/use/migrate-to-9.0.0, it also covers the transition from `.eslintignore` and `.eslintrc.cjs` to the new config format.

**Reply by c0sc0s** [reply]
thx

**Comment by erosman** [comment]
I have recently migrated to v9. 

Create a new file `eslint.config.js`.
Once you copy the basic `export default [ ... ]`, you can then copy/paste the rules from the v8 to v9 config.

More info: [Configuration Files](https://eslint.org/docs/latest/use/configure/configuration-files)

**Reply by c0sc0s** [reply]
thx
